test/fixtures/s3/listObjectsV2.json

```diff
@@ -67,7 +65,7 @@
 			{ "Prefix": "images/dt-with-no-valid-releases/" },
 			{ "Prefix": "images/dt-with-failing-listing/" }
 		],
-		"KeyCount": 55
+		"KeyCount": 57
```

Comment:
This was incorrect to begin with, but all of S3 things should go away once this PR is merged and we enrich the device type model with everything needed.